demo/src/style.css

```diff
@@ -116,7 +116,291 @@ path.nad-arrow-in:not(.nad-state-in .nad-arrow-in) {visibility: hidden}
 .nad-branch-edges .nad-overload .nad-edge-path {animation: line-blink 3s infinite}
 .nad-vl-nodes .nad-overvoltage {animation: node-over-blink 3s infinite}
 .nad-vl-nodes .nad-undervoltage {animation: node-under-blink 3s infinite}
+.nad-zoom-0 {
 
+    .nad-branch-edges .nad-edge-path,
```

Comment:
This is not standard CSS syntax, you need to use pseudo-selector `:is(` instead.